Nos/Views/RepliesView.swift

```diff
@@ -124,33 +131,18 @@ struct RepliesView: View {
                 .onDisappear {
                     relaySubscriptions.removeAll()
                 }
-                VStack {
-                    Spacer()
-                    VStack {
-                        HStack(spacing: 10) {
-                            if let author = currentUser.author {
-                                AvatarView(imageUrl: author.profilePhotoURL, size: 35)
-                            }
-                            ExpandingTextFieldAndSubmitButton(
-                                placeholder: .reply.postAReply,
-                                reply: $reply,
-                                focus: $focusTextView
-                            ) {
-                                await postReply(reply)
-                            }
-                            .onAppear {
-                                focusTextView = showKeyboardOnAppear
-                            }
+                .onAppear {
+                    DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(300)) {
```

Comment:
What do you think about changing this to

```
Task { @MainActor in
    try await Task.sleep(for: .milliseconds(300))
    ...
    
}
```

Reply:
good idea!